e2e/perf/get-baseline-run.py

```
@@ -0,0 +1,119 @@
+
+    data = response.json()
+    runs = data.get('workflow_runs', [])
+
+    if not runs:
+        print("No baseline run found", file=sys.stderr)
+        return None
+
+    # Return the most recent successful run
+    baseline = runs[0]
+
+    print(f"Found baseline run:", file=sys.stderr)
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)